src/paas_charm/_gunicorn/charm.py

```diff
@@ -23,6 +26,67 @@ def _workload_config(self) -> WorkloadConfig:
             framework_name=self._framework_name, unit_name=self.unit.name
         )
 
+    def check_gevent_package(self) -> bool:
+        """Check that gevent is installed.
+
+        Returns:
+            True if gevent is installed.
+        """
+        ddddd = self._container.exec(["pip", "list"])
+        list_output = ddddd.wait_output()[0]
+        return "gevent" in list_output
+
+    def create_webserver_config(self) -> WebserverConfig:
+        """Create a WebserverConfig instance from the charm config.
+
+        Returns:
+            A new WebserverConfig instance.
+
+        Raises:
+            CharmConfigInvalidError: if the charm configuration is not valid.
+        """
+        _webserver_config: WebserverConfig = WebserverConfig.from_charm_config(dict(self.config))
```

Comment:
Variable with underscore in a method is unusual . What is the reason?

Reply:
No reason. Just a complete error.